eth/config.go

```diff
@@ -45,7 +45,7 @@ var DefaultConfig = Config{
 	},
 	NetworkId:          1,
 	LightPeers:         99,
-	LightServ:          0,
+	LightServ:          50,
```

Comment:
If `LightServ` is set to something higher than 0 it will trigger this code in eth/backend.go: 

```
	if s.config.LightServ > 0 {
		if s.config.LightPeers != 0 && s.config.LightPeers >= srvr.MaxPeers {
			return fmt.Errorf("invalid peer config: light peer count (%d) >= total peer count (%d)", s.config.LightPeers, srvr.MaxPeers)
		}
		maxPeers -= s.config.LightPeers
	}

which will ultimately result in a fail with the default run command like this:

```
./bin/geth --datadir /tmp/test/0-node/datadir --syncmode full --debug --port 30303 --rpcvhosts=* --networkid 1101 --verbosity=1 --consoleoutput=stdout --consoleformat=term --nat extip:127.0.0.1 --rpc --rpcport 8545 --rpccorsdomain=* --rpcapi=eth,net,web3,debug,admin,personal,txpool,istanbul --wsorigins=* --ws --wsport 8546 --wsapi=eth,net,web3,debug,admin,personal,txpool,istanbul --mine --minerthreads=10 --nodekeyhex=ad..6c --istanbul.blockperiod 1 --nodiscover --password=/dev/null --unlock=0 --allow-insecure-unlock --ethstats=0-node@localhost:3000 --etherbase=0
Fatal: Error starting protocol stack: invalid peer config: light peer count (99) >= total peer count (50)
```

Reply:
Rsolved in https://github.com/celo-org/celo-blockchain/commit/8521e133dcb5e5631c1243cc35e91c830970e2f1